tests/epyccel/test_epyccel_optional_args.py

```diff
@@ -1,188 +1,190 @@
+        return False
+
+    f = epyccel(f4, language = language)
+
+    # ...
+    assert f(True) == f4(True)
+    assert f() == f4()
+    assert f(None) == f4(None)
+    assert f(False) == f4(False)
+    # ...
+#------------------------------------------------------------------------------
+def test_f5(language):
+    import modules.Module_3 as mod
+
+    modnew = epyccel(mod, language = language)
+
+    # ...
+    assert mod.func(1) == modnew.func(1)
+    assert mod.func() == modnew.func()
+    assert mod.func(None) == modnew.func(None)
+    assert mod.func(0) == modnew.func(0)
+
+#------------------------------------------------------------------------------
+def test_f6(language):
+    import modules.Module_4 as mod
+
+    modnew = epyccel(mod, language = language)
+
+    # ...
+    assert mod.call_optional_1() == modnew.call_optional_1()
+    assert mod.call_optional_2(None) == modnew.call_optional_2(None)
+    assert mod.call_optional_2(0) == modnew.call_optional_2(0)
+    assert mod.call_optional_2() == modnew.call_optional_2()
+#------------------------------------------------------------------------------
+def test_f7(language):
+    import modules.Module_5 as mod
+
+    modnew = epyccel(mod, language = language)
+
+    # ...
+    assert mod.call_optional_1(3) == modnew.call_optional_1(3)
+    assert mod.call_optional_2() == modnew.call_optional_2()
+    assert mod.call_optional_3(3) == modnew.call_optional_3(3)
+
+#------------------------------------------------------------------------------
+def test_f9(language):
+    import modules.Module_5 as mod
+
+    modnew = epyccel(mod, language = language)
+
+    # ...
+    assert mod.call_optional_4(3) == modnew.call_optional_4(3)
+    assert mod.call_optional_5(3) == modnew.call_optional_5(3)
+    assert mod.call_optional_6() == modnew.call_optional_6()
+    assert mod.call_optional_7() == modnew.call_optional_7()
+    assert mod.call_optional_8() == modnew.call_optional_8()
+
+#------------------------------------------------------------------------------
+def test_f10(language):
+    import modules.Module_5 as mod
+
+    modnew = epyccel(mod, language = language)
+
+    # ...
+    assert mod.call_optional_9() == modnew.call_optional_9()
+    assert mod.call_optional_10() == modnew.call_optional_10()
+
+#------------------------------------------------------------------------------
+def test_f11(language):
+    import modules.Module_5 as mod
+
+    modnew = epyccel(mod, language = language)
+
+    # ...
+    assert mod.call_optional_11() == modnew.call_optional_11()
+    assert mod.call_optional_12() == modnew.call_optional_12()
+
+#------------------------------------------------------------------------------
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = [
+            pytest.mark.xfail(reason="Arrays cannot be optional. See #757"),
+            pytest.mark.c]
+        )
+    )
+)
```

Comment:
This error message and the same error message below is the only change in this file. Github is reporting that the whole file changed as my computer changed the line endings from windows format to unix format

Reply:
Who among us is using Windows line endings????

Reply:
Git blame says @MenoIy